### PR TITLE
Move UnitTests

### PR DESCRIPTION
**Pull request by CumulonimbusCalvus**
@peendebak 

I moved the unittests in this PR as requested [here](https://github.com/QuTech-Delft/qtt/pull/518#issuecomment-472173446). I have taken the structure from the demonstator repo's to have consistency.

I still should check whether readthedocs is still working and other additional stuff besides installing, running en unittesting.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,12 +20,12 @@ install:
...
         pip install -r develop_requirements.txt;
       fi
-    - python setup.py develop
+    - pip install -e .
 
 script:
     - ls
     - pytest --version
-    - coverage run --source="./qtt" --omit="*/test_*"  -m pytest
+    - coverage run --source="./src" --omit="*/test_*"  -m pytest
```
Comment (peendebak) [inline .travis.yml]: I guess we can remove the omit part now?
```suggestion
    - coverage run --source="./src"  -m pytest
```

Comment (CumulonimbusCalvus) [reply .travis.yml]: Already done.

**Comment by CumulonimbusCalvus** [comment]
@peendebak 

All OK I think. There is one file (src/qtt/exampledata/misc/P5_y_P5_x.dat) which is changed according to the PR. Still, when I check for diff's there is nothing changed.

**Comment by peendebak** [comment]
@CumulonimbusCalvus Merged. I create a new PR for additional cleanup of the test structure.

**Pull request merged**